modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -123,7 +134,9 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
 
     const userKeyShare = await MPC.keyShare(1, m, n);
     const userGpgKey = await generateGPGKeyPair('secp256k1');
-    const backupGpgKey = await generateGPGKeyPair('secp256k1');
+    const userLocalBackupGpgKey = await generateGPGKeyPair('secp256k1');
+    const krsConstants = await this.getTrustBackupConstants();
+    const thirdPartyBackupPublicGpgKey = await readKey({ armoredKey: krsConstants.trustBackupGPGPubKey });
```

Comment:
We don't usually need to get this Trust backup key. That is only for when we use the trust as KRS.
I don't think the `thirdPartyBackupPublicGpgKey` and `userLocalBackupGpgKey` would usually co-exist.
So, maybe we can at least get this after the `isThirdPartyBackup` and only get this if it is true.

The `createBackupKeyShares` a few lines below may be a okay example to follow to avoid having both `userLocalBackupGpgKey` and `thirdPartyBackupPublicGpgKey` parmeters in every methods that uses the backup public key.

I wish there is a better refactor than even that, though. :/